src/js/pages/app.js

```diff
@@ -54,11 +54,10 @@ class Main extends Component {
     } = this.props;
 
     loadTools(path.join(__dirname, '../../../tC_apps'));
-
-    if (localStorage.getItem('version') !== packageJson.version) {
+    const tcResourcesPath = path.join(ospath.home(), 'translationCore', 'resources');
```

Comment:
Please removed these changes. they are not the reason why tools are loading correctly.